sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/options/ProxyInvocationHandler.java

```diff
@@ -209,6 +267,92 @@ public int hashCode() {
   }
 
   /**
+   * Populate display data. See {@link HasDisplayData#populateDisplayData}. All explicitly-set
+   * pipeline options will be added as display data. Default values are not included unless
```

Comment:
You should drop "default values are not included...". Obvious from above statement